requirements/edx/base.txt

```diff
@@ -108,7 +108,7 @@ edx-milestones==0.3.1     # via -r requirements/edx/base.in
 edx-opaque-keys[django]==2.2.0  # via -r requirements/edx/paver.txt, edx-bulk-grades, edx-ccx-keys, edx-completion, edx-drf-extensions, edx-enterprise, edx-milestones, edx-organizations, edx-proctoring, edx-user-state-client, edx-when, lti-consumer-xblock, xmodule
 edx-organizations==6.9.0  # via -r requirements/edx/base.in
 edx-proctoring-proctortrack==1.0.5  # via -r requirements/edx/base.in
-edx-proctoring==3.8.5     # via -r requirements/edx/base.in, edx-proctoring-proctortrack
+git+https://github.com/edunext/edx-proctoring.git@3.8.5.1#egg=edx-proctoring==3.8.5.1  # via -r requirements/edx/github.in
```

Comment:
I'd prefer if we used 

```
git+https://github.com/edunext/edx-proctoring.git@a19a32232d51d25e2b9725109a3aedecac730fd9#egg=edx-proctoring
```

In other words, install exactly the state as of the revision specified by that SHA, rather than by that tag. (The tag can be removed and reset to a different revision.)

Reply:
Ack, I updated the PR.